src/reformulate.jl

```diff
@@ -76,7 +76,8 @@ function _reformulate_disjunctions(model::JuMP.AbstractModel, method::AbstractRe
     for (idx, disj) in _disjunctions(model)
         disj.constraint.nested && continue #only reformulate top level disjunctions
         dref = DisjunctionRef(model, idx)
-        if requires_exactly1(method) && !haskey(gdp_data(model).exactly1_constraints, dref)
+        if requires_exactly1(method) && !haskey(gdp_data(model).exactly1_constraints, dref) &&
+            !any(has_logical_compliment.(disj.constraint.indicators))
             error("Reformulation method `$method` requires disjunctions where only 1 disjunct is selected, " *
```

Comment:
This error should be updated to note that `exactly1 = false` or no logical compliments are defined for disjunction...